### PR TITLE
E2e scaffolding

### PR DESCRIPTION
**Pull request by NEETweeb**
some e2e scaffolding

### CODE REVIEW

**Review by Aceix — CHANGES_REQUESTED** [review]
fix breaking pr test

**Comment by NEETweeb** [comment]
Hello @Aceix, sir!
The errors can be ignored. That is how 'tests' are declared in Playwright :).

**Comment by NEETweeb** [comment]
@Aceix It appears that there's conflict with Playwright lib and Vite lib in this case. Both have 'test'.
Playwright's 'test' can be implemented even without the 'declare'. While Vite's 'test' has to be inside Vite's 'declare'.

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "buffer": "^6.0.3",
     "classnames": "^2.3.2",
     "dayjs": "^1.11.10",
+    "dotenv": "^16.3.1",
```
Comment (Aceix) [inline package.json:28]: install as dev dependency

**Comment by Aceix** [comment]
> @Aceix It appears that there's conflict with Playwright lib and Vite's lib. In this case, both have 'test'. Playwright's 'test' doesn't need 'declare'. While Vite's 'test' has to be implemented inside Vite's 'declare'.

okay. we'll have to find a way to ignore playwrite from vite then

**Review by Aceix — APPROVED** [review]

**Pull request merged**